src/main/java/net/minecraftforge/common/extensions/IForgeHolderSet.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.common.extensions;
+
+import net.minecraft.core.HolderSet;
+
+public interface IForgeHolderSet<T>
+{
+    default public HolderSet<T> self()
```

Comment:
The `self()` method in extension interfaces should be `private`.
```suggestion
    private HolderSet<T> self()
```